src/components/CityOverviewContainer/CityOverview/CityOverview.js

```diff
@@ -6,6 +6,7 @@ import PageWrapper from '../../PageWrapper/PageWrapper';
 import {
   HERO_INDICATOR_ID,
   HERO_INDICATOR_CATEGORY,
+  // HERO_INDICATOR_COLOR_NAME,
```

Comment:
What's happened here? Is `HERO_INDICATOR_COLOR_NAME` is no longer used? It's now hard coded as `infrastructure` but that's not right is it? Shouldn't it be planning?

Reply:
Whoops, I was testing something and forgot to change back. Well spotted.